tests/core/plan-env-file/data/plan.fmf

```diff
@@ -0,0 +1,40 @@
+/test:
```

Comment:
Let's call the file `main.fmf` as `plan` becomes part of the `fmf` name:

```
Found 1 test: /plan/test.
Found 4 plans: /plan/plan/env-file-override, /plan/plan/execute-override, /plan/plan/multiple and /plan/plan/single.
```

Reply:
Fixed in 9ecf336.